lib/ledgerEventStorage.js

```diff
@@ -294,34 +295,51 @@ module.exports = class LedgerEventStorage {
    * Gets the latest configuration event that has consensus.
    *
    * options - a set of options used when retrieving the event.
+   *   [beforeBlockHeight] - the latest config before the specified blockHeight.
```

Comment:
This seems a little awkward. Can we do "blockHeight" instead and the caller can do `blockHeight - 1` as needed? It makes more sense to me to ask for the latest config at a certain block height, rather than the config from before that. I think we're letting too much of our particular use case bleed into the API design here.

Reply:
Agreed it is weird, get latest before is definitely confusing.

The point of this API in my opinion is to determine what the active configuration is for a particular blockHeight.  I don't want to do any fancy math.  I want to find the blockHeight on an event and pass that into this thing to get the relevant config.

Perhaps a different API would be more appropriate.  What about:
```
getActiveConfig({blockHeight}, callback);
```

Reply:
@mattcollier,

Yes, let's do `getActiveConfig`. It gives you the config to use for a particular `blockHeight`, I like that.